firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java

```diff
@@ -351,6 +372,10 @@ private boolean shouldFetchAndActivateRemoteConfigValues() {
         > TIME_AFTER_WHICH_A_FETCH_IS_CONSIDERED_STALE_MS;
   }
 
+  private boolean passesRandomAppStartDelay() {
```

Comment:
Though a private method, can we add some documentation about what we can expect from that method?

Reply:
done